c2cgeoportal/scripts/c2ctool.py

```diff
@@ -214,10 +214,29 @@ def get_project():
             return yaml.safe_load(f)
 
     def test_checkers(self):
-        http = httplib2.Http()
+        # Read project options for checker http session connection
+        try:
+            checker_protocol = self.project["checker_protocol"]
+        except KeyError:
+            checker_protocol = "http"
+            print("No checker_protocol defined in project, using default value {}".format(checker_protocol))
```

Comment:
Don't use an exception handling just to add a massage, just display the URL in any cases...

Reply:
The goal of the exception handling here is not to add a message, but to provide a default value in case the configuration option "checker_protocol" is not present in the project.

Reply:
=> `checker_protocol = self.project.get("checker_protocol", "http")` :-)